etc/config/.env.example

```diff
@@ -43,7 +43,7 @@ BROWSER=chrome
 #DEFAULT_TIMEZONE=America/Los_Angeles
 
 #*** These properties impact the modules loaded into MFTF, you can point to your own full path, or a custom set of modules located with the core set
-MODULE_WHITELIST=Magento_Framework,Magento_ConfigurableProductWishlist,Magento_ConfigurableProductCatalogSearch
+MODULE_WHITELIST=Magento_Framework,ConfigurableProductWishlist,ConfigurableProductCatalogSearch
```

Comment:
This are plain white list module paths. They should not be assumed to be magento module to match based on current logic

Reply:
From my reviewing it seems `Magento_Module` format is still usable for `.env` since it merges to an existing list of `Magento_Module` formatted modules.

@jilu1 do either format work based on your implementation? I think the biggest concern here is backward compatibility.

Reply:
Before the change, these modules are not found by the ModuleResolver because they did not follow the search pattern.
After change, we search this path specifically. And they are in white list so it will be included.